views/src/Pages/ForgotPassword.jsx

```diff
@@ -0,0 +1,208 @@
+import React, { useEffect, useState, useRef } from "react";
+import {
+  Stepper,
+  Button,
+  Group,
+  TextInput,
+  Container,
+  PinInput,
+} from "@mantine/core";
+import axios from "axios";
+
+import { baseUrl } from "../API/index";
+
+
+
+function ForgotPassword() {
+  const [active, setActive] = useState(0);
+  const [email, setEmail] = useState("");
+  const [emailSubmitted, setEmailSubmitted] = useState(false);
+  const [otpValidated, setOtpValidated] = useState(false);
+  const errorMsg = useRef(null);
+  const [otp, setOtp] = useState(0);
+  const [newPwd, setNewPwd] = useState(null);
+  const [confirmPwd, setConfirmPwd] = useState(null);
+  const updatepasswordValidator = (input, confirmation) => {
```

Comment:
This function checks but only sets the error message for the first failing condition. I'd look into accumulating the error.

Reply:
Applied cumulative error checking and multiple error display using state arrays